stuff.js

```diff
@@ -0,0 +1,36 @@
+// uploads image to AWS s3 and updates user schema with url of profile image
+router.post("/profileImageUpload", async function (req, res, next) {
+  if (req.files === null) {
+    return res.status(400).json({ msg: "No file uploaded" });
+  }
+  const { id } = req.user;
+
+  // upload image to s3 with the help of multer-s3
+  profileImgUpload(req, res, (error) => {
+    if (error) {
+      return res.status(400).json({
+        errors: {
```

Comment:
should we make errors an array like elsewhere in the code-base

Reply:
Great catch.